dhis-2/dhis-services/dhis-service-tracker/src/test/resources/tracker/validations/events-with_valid_option_value.json

```diff
@@ -22,7 +22,7 @@
           "storedBy": "admin",
           "createdAt": "2019-01-28T12:10:38.113",
           "dataElement": "LjQPcfuEI6S",
-          "value": "male",
```

Comment:
was this value previously invalid?

Reply:
no it was considered valid. but last time we decided to not use equalsIgnoreCase.